webapp/src/main/java/com/box/l10n/mojito/security/RequestLoggingConfig.java

```diff
@@ -1,15 +1,12 @@
 package com.box.l10n.mojito.security;
 
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.web.filter.CommonsRequestLoggingFilter;
+import org.springframework.stereotype.Component;
 
-@Configuration
+@Component
```

Comment:
Why the change from @Config to @Component?

Does @ConfigurationProperties suit better here, that way you can set the property prefix once and drop the @Value annotations, see `SlackClientsConfigurationProperties` for an example

Reply:
https://github.com/pinterest/mojito/blob/master/common/src/main/java/com/box/l10n/mojito/slack/SlackClientsConfigurationProperties.java

Reply:
Regarding the switch to `@Component`:
> @Configuration is specifically for creating beans via methods within the class, while @Component is for marking classes to be automatically picked up by Spring's component scanning.

This feels more in-line with the class' focus now.

I can use the @ConfigurationProperties though. It does clean things up a bit